test-packages/macro-fastboot-tests.test.js

```diff
@@ -0,0 +1,37 @@
+const execa = require('execa');
+const FastBoot = require('fastboot');
+
+test('FastBoot - macro-test app boots in FastBoot environment using custom app-boot', async () => {
+  jest.setTimeout(120000);
+
+  const buildProcess = execa('ember', ['build'], {
+    cwd: `${__dirname}/macro-tests`,
+  });
+
+  const buildCompletePromise = new Promise(res => {
```

Comment:
wouldn't `await execa('ember', ['build'], { ... }` also tell us the build is complete, at which point we know `dirname + '/macro-tests/dists'` is ready to be consumed by fastboot

Reply:
I cleaned up the code use await on the build command.